app/Console/Command/FetchMovies.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use Broadway\Repository\Repository;
+use Broadway\UuidGenerator\UuidGeneratorInterface;
+use CultuurNet\UDB3\Calendar\Calendar as LegacyCalendar;
+use CultuurNet\UDB3\Event\Event as EventAggregate;
+use CultuurNet\UDB3\Event\EventType;
+use CultuurNet\UDB3\Kinepolis\KinepolisService;
+use CultuurNet\UDB3\Kinepolis\MovieMappingRepository;
+use CultuurNet\UDB3\Language as LegacyLanguage;
+use CultuurNet\UDB3\Kinepolis\ParsedMovie;
+use CultuurNet\UDB3\Offer\Commands\UpdateCalendar;
+use CultuurNet\UDB3\Event\Commands\UpdateDescription;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class FetchMovies extends AbstractCommand
```

Comment:
I would make the name again more specific, this can certainly refer to Kinepolis